src/SDK.ts

```diff
@@ -249,9 +249,10 @@ export default class SDK implements ISDK {
     addOcUserAgentHeader(this.ocUserAgent, requestHeaders);
     this.setSessionIdHeader(this.sessionId, requestHeaders);
 
-    if (!this.configuration.useUnauthReconnectIdSigQueryParam) {
-      // Append reconnect id on the endpoint if vailable
-      if (reconnectId) {
+    // If should only be applicable on unauth chat & the flag enabled
+    const shouldUseSigQueryParam = !authenticatedUserToken && this.configuration.useUnauthReconnectIdSigQueryParam === true;
```

Comment:
do we need to initialize useUnauthReconnectIdSigQueryParam to false in the default config?

Reply:
yes, it will be enabled in upstream components instead.